### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by taylor-bsg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,11 +78,12 @@ to the filesystem using normal calls:
     }
 
 ### Building Linux
-To build a SMP Linux executable for BlackParrot:
+To build a SMP Linux executable for BlackParrot (make sure first to follow the above instructions for building the SDK):
 ```
 make -j linux OPENSBI_NCPUS=<n>
+./install/bin/dromajo --host --ncpus=8 linux/linux.riscv # verify linux runs on the dromajo simulator
```
Comment (farzamgl) [inline README.md:84]: `./install/bin/dromajo --host --ncpus=<n> prog/linux/linux.riscv`

`linux/linux.riscv` is also correct, but we should probably refer to the prog directory here.

Comment (dpetrisko) [reply README.md:84]: I think linux/linux.riscv is better because they might be building in that directory locally for whatever reason

**Review by dpetrisko — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (farzamgl) [reply README.md:84]: I don't feel strongly about this either way.
Have to fix the `--ncpus` thing before merging though.

Comment (dpetrisko) [reply README.md]: fixed

**Comment by taylor-bsg** [comment]
Merge at will

**Pull request merged**